assets/dev/js/editor/components/hotkeys/component.js

```diff
@@ -9,7 +9,7 @@ export default class Component extends ComponentModalBase {
 	defaultShortcuts() {
 		return {
 			'': {
-				keys: 'ctrl+?',
+				keys: 'ctrl+?, shift+?',
```

Comment:
It should somehow be reflected in the cheatsheet that `ctrl+?` is also supported, as well as in our doc. Otherwise, users might complain about this unexpected behavior.

Reply:
It's deprecated.